### ISSUE TITLE
Failed at the s2geometry-node@1.2.3 install script 'node-gyp rebuild'

### ISSUE THREAD

**Issue by klisman** [Armax/Pokemon-GO-node-api#90]
I got this trying to install the api, any idea what should be?
Thanks.

```
C:\Users\Usuario\pokegowebdex>npm install pokemon-go-node-api
> s2geometry-node@1.2.3 install C:\Users\Usuario\pokegowebdex\node_modules\s2geometry-node
> node-gyp rebuild
C:\Users\Usuario\pokegowebdex\node_modules\s2geometry-node>if not defined npm_config_node_gyp (node "C:\Users\Usuario\AppData\Roaming\npm\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (node "" rebuild )
gyp ERR! build error
gyp ERR! stack Error: EPERM: operation not permitted, open 'C:\Users\Usuario\.node-gyp\4.4.5\Release\node.lib'
gyp ERR! stack     at Error (native)
gyp ERR! System Windows_NT 10.0.10586
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\Usuario\\AppData\\Roaming\\npm\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd C:\Users\Usuario\pokegowebdex\node_modules\s2geometry-node
gyp ERR! node -v v4.4.5
gyp ERR! node-gyp -v v3.4.0
gyp ERR! not ok
pokegowebdex@0.0.0 C:\Users\Usuario\pokegowebdex
`-- (empty)

npm ERR! Windows_NT 10.0.10586
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\Usuario\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "install" "pokemon-go-node-api"
npm ERR! node v4.4.5
npm ERR! npm  v3.10.5
npm ERR! code ELIFECYCLE

npm ERR! s2geometry-node@1.2.3 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the s2geometry-node@1.2.3 install script 'node-gyp rebuild'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the s2geometry-node package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs s2geometry-node
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls s2geometry-node
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\Usuario\pokegowebdex\npm-debug.log
npm ERR! code 1
```

**Comment by klisman** [comment]
s2geometry-node needs a fix for windows platform. Waiting the Pull approval.

**Comment by jimdubbs** [comment]
Looks like this PR has been approved.

**Comment by coolaj86** [comment]
Fixed in #165 by using pure JavaScript - no C++ or compiler necessary.
